tests/test_models.py

```diff
@@ -1,38 +1,186 @@
 """
-Test cases for YourResourceModel Model
+Test cases for Wishlist Model
 
 """
 import os
 import logging
 import unittest
-from service.models import YourResourceModel, DataValidationError, db
+from service import app
+from service.models import Wishlist, DataValidationError, db
+from tests.factories import WishlistFactory
 
 
+DATABASE_URI = os.getenv(
+    "DATABASE_URI", "postgresql://postgres:postgres@localhost:5432/postgres"
+)
+
 ######################################################################
-#  YourResourceModel   M O D E L   T E S T   C A S E S
+#  Wishlist   M O D E L   T E S T   C A S E S
 ######################################################################
 # pylint: disable=too-many-public-methods
-class TestYourResourceModel(unittest.TestCase):
-    """ Test Cases for YourResourceModel Model """
+class TestWishlist(unittest.TestCase):
+    """ Test Cases for Wishlist Model """
 
     @classmethod
     def setUpClass(cls):
         """ This runs once before the entire test suite """
+        app.config["TESTING"] = True
+        app.config["DEBUG"] = False
+        app.config["SQLALCHEMY_DATABASE_URI"] = DATABASE_URI
+        app.logger.setLevel(logging.CRITICAL)
+        Wishlist.init_db(app)
 
     @classmethod
     def tearDownClass(cls):
         """ This runs once after the entire test suite """
+        db.session.query(Wishlist).delete()  # clean up the last tests
+        db.session.commit()
+        #db.drop_all() #Drops all tables if needed for updating schema
 
     def setUp(self):
         """ This runs before each test """
+        db.session.query(Wishlist).delete()  # clean up the last tests
+        db.session.commit()
 
     def tearDown(self):
         """ This runs after each test """
+        db.session.remove()
 
     ######################################################################
     #  T E S T   C A S E S
     ######################################################################
 
-    def test_example_replace_this(self):
-        """ It should always be true """
-        self.assertTrue(True)
+    def test_create_a_wishlist(self):
```

Comment:
Maybe name this test_wishlist_constructor?  I found it a little confusing to call this 'create' even though it's not Create in the CRUD sense since we are only creating an instance in memory.

Reply:
Got it, I based it on the style/code the professor linked us with. 
I agree it tests the Wishlist Model constuctor, it functionally validates the data entries of the model is what we expect.
The latter "add a wishlist" tests both creating and adding into a wishlist, hence creation of an entry in the db. 

Would it be okay to rename this one to "define_a_wishlist" then?
And the latter one to "create_a_wishlist"?